pytest/tests/replay/replay.py

```diff
@@ -0,0 +1,161 @@
+        py_tx.args = base64.b64decode(py_tx.args.encode('ascii'))
+        py_tx.deposit = int(py_tx.deposit)
+    elif tx_type == "Transfer":
+        py_tx.deposit = int(py_tx.deposit)
+    elif tx_type == "Stake":
+        py_tx.stake = int(py_tx.stake)
+        py_tx.publicKey = convert_dumped_public_key_to_py_public_key(py_tx.publicKey)
+    elif tx_type == "AddKey":
+        raise ValueError('Unsupported tx type: %s' % tx_type)
+    elif tx_type == "DeleteKey":
+        py_tx.publicKey = convert_dumped_public_key_to_py_public_key(py_tx.publicKey)
+    else:
+        raise ValueError('Unknown tx type: %s' % tx_type)
+    return py_tx
+
+def convert_dumped_action_to_py_action(action_dict):
+    assert(len(action_dict) == 1)
+    for tx_type, value in action_dict.items():
+        contents = convert_dumped_rust_instance_to_py_object(value, convert_transaction_type_string_to_class(tx_type))
+        contents = fix_dumped_fields_by_tx_type(contents, tx_type)
+        action = Action()
+        action.enum = tx_type[0].lower() + tx_type[1:]
+        setattr(action, action.enum, contents)
+        return action
+
+def send_resigned_transactions(tx_path, key_pair):
+    LOCALHOST = '127.0.0.1'
+    base_block_hash = mocknet_helpers.get_latest_block_hash(addr=LOCALHOST)
+    my_account = Account(key_pair,
+                         init_nonce=0,
+                         base_block_hash=base_block_hash,
+                         rpc_infos=[(LOCALHOST, "3030")])
+
+    schema = dict(tx_schema + crypto_schema + bridge_schema)
+    with open(tx_path) as fin:
+        txs = json.load(fin, object_pairs_hook=OrderedDict)
+    for original_signed_tx in txs:
+        tx = convert_dumped_rust_instance_to_py_object(original_signed_tx['transaction'], Transaction)
+        if hasattr(tx, 'blockHash'):
+            tx.blockHash = base_block_hash
+        if hasattr(tx, 'actions'):
+            tx.actions = [convert_dumped_action_to_py_action(action_dict) for action_dict in tx.actions]
+        tx.publicKey = PublicKey()
+        tx.publicKey.keyType = 0
+        tx.publicKey.data = key_pair.decoded_pk()
+        msg = BinarySerializer(schema).serialize(tx)
+        hash_ = hashlib.sha256(msg).digest()
+        signature = Signature()
+        signature.keyType = 0
+        signature.data = key_pair.sign_bytes(hash_)
+        resigned_tx = SignedTransaction()
+        resigned_tx.transaction = tx
+        resigned_tx.signature = signature
+        resigned_tx.hash = hash_
+        my_account.send_tx(BinarySerializer(schema).serialize(resigned_tx))
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Setup replay')
+    parser.add_argument('--tx-json', type=str, required=True, help="Path of tx history json")
+    parser.add_argument('--genesis', type=str, required=True, help="Path of genesis")
+    parser.add_argument('--output-dir', type=str, required=True, help="Path of the new home directory")
+    args = parser.parse_args()
+
```

Comment:
I'd add a 'command' parameter -- so that you can run this binary and either 'generate genesis & configs' or 'resign & send' .

this would make it more 'scriptable' (as with the current approach with 'waiting on enter' - it is harder to do).

Then the shell could run:

cargo build -p neard
replay.py generate_genesis /tmp/foo
neard run /tmp/foo &
replay.py sign_and_send